.github/workflows/pythonpackage.yml

```
@@ -0,0 +1,41 @@
+name: Python 3 testing
+
+on: [push, pull, pull_request]
```

```suggestion
on: [push, pull_request]
```

`pull` is not a valid event. Maybe that's why it didn't start a run.

---

The previous comment was without "__pull__" but no test ran.